assets/js/blocks/customer-account/style.scss

```diff
@@ -0,0 +1,16 @@
+.wp-block-woocommerce-customer-account {
+	a {
+		text-decoration: none;
+		align-items: center;
+		display: flex;
+		color: currentColor !important;
+
+		&:hover {
+			text-decoration: underline;
+		}
+
+		.icon + .label {
+			margin-left: $gap-smaller;
+		}
+	}
+}
```

Comment:
Hey, you are right, this is solved on this other PR, since Viv also mentioned this issue https://github.com/woocommerce/woocommerce-blocks/pull/7944
I'll merge both at the same time so this won't be an issue.